api/webhook.go

```diff
@@ -129,32 +133,47 @@ func PostWebhook(c *gin.Context) {
 		return
 	}
 
-	h, r, b := webhook.Hook, webhook.Repo, webhook.Build
+	h, repo, b := webhook.Hook, webhook.Repo, webhook.Build
```

Comment:
I'll flip flop

Reply:
Flipping them creates a mirror "off feeling" in `publishToQueue` call: 
```go
	go publishToQueue(
		queue.FromGinContext(c),
		database.FromContext(c),
		p,
		b,
		repo,
		u,
	)
```

Not really sure how to proceed. The reason we can't just use `r` is because we overwrite what the SCM provides for repository information.

Reply:
i see, you need access to the original repo parsed out of the hook payload to access topics and branch. other than saving it off into its own variables and returning to shadowing `r`, options are limited, save for a rewrite. i guess for now, it doesn't matter which way they are flipped. sorry for the alarm :(